src/lib/core/a11y/list-key-manager.ts

```diff
@@ -0,0 +1,64 @@
+import {EventEmitter, Output, QueryList} from '@angular/core';
+import {UP_ARROW, DOWN_ARROW, TAB} from '../core';
+
+export interface MdFocusable {
+  focus(): void;
+  disabled: boolean;
+}
+
+export class ListKeyManager {
```

Comment:
We should have unit tests for just this class (with some fake `MdFocusable` items)